action.yml

```
@@ -351,6 +355,12 @@ runs:
         key: cvmfs-apt-cache-${{ steps.lsb-release.outputs.id-release }}-${{ steps.lsb-release.outputs.arch }}-${{ hashFiles('action.yml') }}
         path: |
           ${{ inputs.apt_cache }}
+    - uses: actions/cache@v4
+      if: ${{ inputs.actions_cache == 'true' }}
+      with:
+        key: cvmfs-cache-${{ steps.lsb-release.outputs.id-release }}-${{ steps.lsb-release.outputs.arch }}-${{ hashFiles('action.yml') }}
+        path: |
+          ${{ inputs.cvmfs_cache_base }}
```

```suggestion
          ${{ inputs.cvmfs_cache_base }}
```
One further issue may be that the cache directory is used for a number of locks and marker files that we may not want to cache. I think in the end you only want to keep the actual data files in the hash directories ./00/ ... ./ff and maybe the ./cachedb file.

I'll  test a few configurations to see what works best.